*javax.activity.ActivityCompletedException* *ActivityCompletedException* This ex

public class ActivityCompletedException
  extends    |java.rmi.RemoteException|

|javax.activity.ActivityCompletedException_Description|
|javax.activity.ActivityCompletedException_Fields|
|javax.activity.ActivityCompletedException_Constructors|
|javax.activity.ActivityCompletedException_Methods|

================================================================================

*javax.activity.ActivityCompletedException_Constructors*
|javax.activity.ActivityCompletedException()|Constructs a new instance with nul
|javax.activity.ActivityCompletedException(String)|Constructs a new instance wi
|javax.activity.ActivityCompletedException(String,Throwable)|Constructs a new t
|javax.activity.ActivityCompletedException(Throwable)|Constructs a new throwabl

*javax.activity.ActivityCompletedException_Description*

This exception may be thrown on any method for which Activity context is 
accessed and indicates that ongoing work within the Activity is not possible. 
This may be because the Activity has been instructed to complete with 
CompletionStatusFailOnly or has ended as a result of a timeout. This exception 
will be propagated across ORB boundaries via an 
org.omg.CORBA.ACTIVITY_COMPLETED system exception. An application should handle 
this error by attempting to complete the Activity. 


*javax.activity.ActivityCompletedException()*

public ActivityCompletedException()

Constructs a new instance with null as its detail message. 


*javax.activity.ActivityCompletedException(String)*

public ActivityCompletedException(java.lang.String message)

Constructs a new instance with the specified detail message. 

    message - the detail message. 

*javax.activity.ActivityCompletedException(String,Throwable)*

public ActivityCompletedException(
  java.lang.String message,
  java.lang.Throwable cause)

Constructs a new throwable with the specified detail message and cause. 

    message - the detail message. 
    cause - a chained exception of type Throwable. 

*javax.activity.ActivityCompletedException(Throwable)*

public ActivityCompletedException(java.lang.Throwable cause)

Constructs a new throwable with the specified cause. 

    cause - a chained exception of type Throwable. 

